File: 2024-2/0682/unidad06.d_Web_04/SoftPub/SoftPubGrupoTest/GrupoBoTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPubGrupoBussiness;

namespace SoftPubGrupoTest
{
    internal class GrupoBoTest
    {
        private static GrupoBo grupoBo;

        public static void testGrupoBo()
        {
            System.Console.WriteLine("\ntestGrupoBo");
            grupoBo = new GrupoBo();
            testGrupoBoInsertar();
            System.Console.ReadLine();
        }

        private static void testGrupoBoInsertar()
        {
            int? id = grupoBo.insertar("Grupo de Inteligencia Artificial PUCP", "IA-PUCP");
            System.Console.WriteLine("idGrupo: " + id);
        }
    }
}

File: 2024-2/Proyecto DAO SoftPub Java/SoftPubGrupoTest/src/softpubgrupotest/GrupoBoTest.java
package softpubgrupotest;

import java.util.ArrayList;
import pe.edu.pucp.softpub.grupo.bo.GrupoBO;

class GrupoBoTest {
    
    private static GrupoBO grupoBo;

    static void testGrupoBo() {
        System.out.println("\ntestGrupoBo");
        grupoBo = new GrupoBO();
        ArrayList<Integer> listaId = new ArrayList<>();
        testGrupoBoInsertar(listaId);
    }

    private static void testGrupoBoInsertar(ArrayList<Integer> listaId) {
        Integer id = grupoBo.insertar("Grupo de Inteligencia Artificial PUCP", "IA-PUCP");
        System.out.println("idGrupo: " + id);
        listaId.add(id);
    }

}
